README.md

```diff
@@ -280,23 +280,87 @@ Restart the daemon normal loop cleanly with:
 sudo  wifi-connect start
 ```
 
-# Unit Tests
+# Tests
+## Unit Tests
 
 You can run all tests by executing 
-	go test -v ./...
+```bash
+go test -v ./...
+```
 or
-	./run-check --unit
-
+```bash
+./run-check --unit
+```
 
 In order to run specific package test, you can:
-	go test -v ./<package>
+```bash
+go test -v ./<package>
+```
 
 To run a specific test:
-	go test -v -run <testname> 
+```bash
+go test -v -run <testname> 
+```
 
 For example:
-	go test -v ./wifiap
-	go test -v ./wifiap -run TestShow
+```bash
+go test -v ./wifiap
+go test -v ./wifiap -run TestShow
+```
 
 More info in https://golang.org/pkg/testing
 
+## Spread tests
+
+We have a set of spread (https://github.com/snapcore/spread) tests which
+can be executed on a virtual machine or real hardware.
+
+In order to run those tests you need the follow things
+
+* ubuntu-image
+* spread
+
+ You can install ubuntu-image snap
+
+```bash
+snap install --edge --devmode ubuntu-image
```

Comment:
for me, ubuntu-image only installs from edge with --classic (not --devmode) today:
$ snap install --edge --classic ubuntu-image
ubuntu-image (edge) 1.0+snap2 from 'canonical' installed